### PR TITLE
add ZOOM sanitizer

### PR DESCRIPTION
**Pull request by skyzh**

### CODE REVIEW

File: sanitizer.py [hunk 1]
```diff
@@ -1,18 +1,23 @@
 import json
+import re
 
 from os import walk
 from pathlib import Path
 
+
 def process_json(filename):
-    obj = json.loads(Path(filename).read_text(), encoding="utf-8")
```
Comment (danyang685) [inline sanitizer.py]: @skyzh, you mistyped this line.

File: sanitizer.py [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
+    obj = json.loads(Path(filename).read_text(encoding="utf-8"))
     cnt = {}
     for key in range(len(obj)):
         for data_key in obj[key].keys():
-            if isinstance(obj[key][data_key], str) and "zoom" in obj[key][data_key].lower():
-                obj[key][data_key] = "<ZOOM info masked>"
-                cnt[data_key] = cnt.get(data_key, 0) + 1
+            if data_key == 'xkbz' and isinstance(obj[key][data_key], str):
+                raw_content = obj[key][data_key]
+                obj[key][data_key] = re.sub(r'([0-9]{7,})', 'XXXXXXXX', raw_content)
```
Comment (danyang685) [inline sanitizer.py:15]: Now it replaces number longer than 7 digits into `XXXXXXXX`

File: sanitizer.py [hunk 3]
```diff
@@ -1,18 +1,23 @@
...
+                if obj[key][data_key] != raw_content:
+                    cnt[data_key] = cnt.get(data_key, 0) + 1
     print(cnt)
-    Path(filename).write_text(json.dumps(obj, ensure_ascii=False, indent=True), encoding="utf-8")
+    Path(filename).write_text(json.dumps(obj, ensure_ascii=False, indent=1), encoding="utf-8")
```
Comment (danyang685) [inline sanitizer.py:19]: Indent should an integer.

**Review by danyang685 — APPROVED** [review]

File: sanitizer.py (hunk 3 above)
Comment (skyzh) [reply sanitizer.py:19]: You may have to fix the crawler, where I just copied `indent=True` from...

File: sanitizer.py (hunk 2 above)
Comment (skyzh) [reply sanitizer.py:15]: Thanks!

**Pull request merged**